src/util/ImageUploader.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+class QJsonObject;
+
+namespace chatterino {
+
+class Settings;
+
+namespace imageuploader {
+namespace detail {
+
+QJsonObject exportSettings(Settings &s);
```

Comment:
Make this take a `const Settings &s` instead, to clarify that we're not looking to make any modifications to the settings in this function.